.github/workflows/package-windows.yml

```diff
@@ -0,0 +1,123 @@
+name: package-windows
+
+# This workflow will be triggered on:
+# - push on the main branch
+# - push on the releases branches
+# - tag push matching the version pattern
+on:
+  pull_request:
+    paths:
+      - .github/workflows/package-windows.yml
+  push:
+    branches:
+      - master
+      - "[0-9]+.[0-9]+"
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+
+env:
+  python-version: "3.9"
+
+jobs:
+  package-windows-build-wheel:
+    runs-on: windows-2022
+    env:
+      # Need feature from poetry 1.2.x that aren't present in 1.1.x
+      poetry-version: 1.2.0b3
+    outputs:
+      wheel-version: ${{ steps.wheel-version.outputs.version }}
+    steps:
+      - name: Install poetry
+        shell: bash
+        run: |
+          export POETRY_HOME=${APPDATA}/.poetry
```

Comment:
`set -eux` should be added  at the beginning of every bash script ;-)

Reply:
done